src/Views/Games/2048/Instructions.js

```diff
@@ -14,7 +14,17 @@ export default () => {
 
     return (
         <div className={clsx(classes.root)}>
-
+        <p>
+        The goal of 2048 is to create a tile with the value 2048, which is done by combining tiles that have
+        the same number. This can be done by pressing the arrow keys on your keyboard to slide all tiles
+        in the designated direction. When two tiles of the same number collide, they will merge together 
+        with the combined value of the two tiles. After every move, a new tile will randomly appear with a 
```

Comment:
This will read better if it's broken up into sections and put into a column. See if you can find an easy way to logically segment this a bit.